css/desktop.css

```diff
@@ -0,0 +1,571 @@
+@media screen and (min-width: 992px) {
+  body {
+    background-color: #e5e5e5;
+    font-family: Poppins, sans-serif;
+    padding: 0;
+    margin: 0;
+  }
+
+  a {
+    text-decoration: none;
+    text-decoration-line: none;
+  }
+
+  .flex-item1,
+  .shape,
+  .item,
+  .flex-item2,
+  .mobile-item,
+  .mobile {
+    display: none;
+  }
```

Comment:
- [ ] KIndly this contact form has a background. let's work on it so that we may go to the next project. check here 👇 
![image](https://user-images.githubusercontent.com/65068771/150535093-42c8c30b-6f80-4ce6-90cc-2ec336ac3141.png)